replication-api-impl/src/test/java/org/codice/ditto/replication/api/impl/ReplicatorStoreFactoryImplTest.java

```diff
@@ -81,10 +89,54 @@ public void setUp() throws Exception {
     factory.setCatalogFramework(framework);
   }
 
+  @After
+  public void tearDown() throws Exception {
+    System.clearProperty("replication.connection.timeout");
```

Comment:
✏️ Please use the `RestoreSystemProperties` JUnit rule or its annotation instead